website/docs/r/catalog_media.html.markdown

```diff
@@ -45,3 +45,45 @@ The following arguments are supported:
 * `upload_piece_size` - (Optional) - size in MB for splitting upload size. It can possibly impact upload performance. Default 1MB.
 * `show_upload_progress` - (Optional) - Default false. Allows to see upload progress
 * `metadata` - (Optional; *v2.5+*) Key value map of metadata to assign
+
+## Attribute reference
+
+Supported in provider *v2.5+*
+
+* `is_iso` - (Computed) returns True if this media file is ISO
+* `owner_name` - (Computed) returns owner name
+* `is_published` - (Computed) returns True if this media file is in a published catalog
+* `creation_date` - (Computed) returns creation date
+* `size` - (Computed) returns media storage in Bytes
+* `status` - (Computed) returns media status
+* `storage_profile_name` - (Computed) returns storage profile name
+
+## Importing
+
+Supported in provider *v2.5+*
+
+~> **Note:** The current implementation of Terraform import can only import resources into the state. It does not generate
+configuration. [More information.][docs-import]
+
+An existing media item can be [imported][docs-import] into this resource via supplying its path.
+The path for this resource is made of orgName.catalogName.mediaName.
```

Comment:
I suggest to use notion with dashes everywhere. I.e. "org-name.catalog-name.media-name" - it's just more consistent and easier to scan with eyes.

Reply:
fixed

Reply:
Thanks, just one more ask: there's a dot at the end of the sentence, which may trip inexperienced user, please remove it as well.

Reply:
done